### PR TITLE
Removing redirect url

### PR DESCRIPTION
**Pull request by gaetbout**
After looking in the (thin) ASDF documentation I couldn't find any way to tell the bash script when an installation is from asdf or not 🙁 

Hope this solves issue #11

### CODE REVIEW

File: bin/latest-stable [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -euo pipefail
 
 current_script_path=${BASH_SOURCE[0]}
+printf ${BASH_SOURCE[0]}
```
Comment (mkaput) [inline bin/latest-stable:6]:
```suggestion
```

I think this has slipped through?

**Review by mkaput — COMMENTED** [review]
Thank you a lot for investigating this! Sad to see that it's impossible, but I really appreciate the effort trying to think of the solution ❤️

File: bin/latest-stable (hunk 1 above)
Comment (gaetbout) [reply bin/latest-stable]: image

Comment (gaetbout) [reply bin/latest-stable]: image

Comment (gaetbout) [reply bin/latest-stable]: Woopsie, should be fine now :)

**Review by mkaput — APPROVED** [review]

**Comment by mkaput** [comment]
❤️❤️❤️❤️❤️

**Pull request merged**